lib/talkable/configuration.rb

```diff
@@ -30,6 +35,19 @@ def js_integration_library
       @js_integration_library || default_js_integration_library
     end
 
+    def clean
+      self.site_slug    = nil
+      self.api_key      = nil
+      self.server       = DEFAULT_SERVER
+      self.read_timeout = 60
+      self.open_timeout = 60
```

Comment:
I would introduce something like `default_configuration` and use it in `clean` and `initialize`

Reply:
In `initialize` we have
```
self.site_slug    = ENV["TALKABLE_SITE_SLUG"]
self.api_key      = ENV["TALKABLE_API_KEY"]
self.server       = DEFAULT_SERVER
self.read_timeout = DEFAULT_TIMEOUT
self.open_timeout = DEFAULT_TIMEOUT
```
in `clean`
```
self.site_slug    = nil
self.api_key      = nil
self.server       = DEFAULT_SERVER
self.read_timeout = DEFAULT_TIMEOUT
self.open_timeout = DEFAULT_TIMEOUT
```
Do you want to make `site_slug` and `api_key` as `nil` or as `ENV`?

Reply:
I would keep it consistent and use `ENV` everywhere. And having everything in one place (aka `default_configuration`) is still preferable.